packages/gateway/scripts/denylist.js

```diff
@@ -0,0 +1,149 @@
+import fetch from '@web-std/fetch'
+import path from 'path'
+import toml from 'toml'
+import { CID } from 'multiformats'
+import { base32 } from 'multiformats/bases/base32'
+import { sha256 } from 'multiformats/hashes/sha2'
+import fs from 'fs'
+import * as uint8arrays from 'uint8arrays'
+
+/**
+ * @typedef {{ id: string, title: string }} Namespace
+ * @typedef {{ name: string, metadata: any }} Key
+ * @typedef {{ key: string, value: any, metadata?: any }} BulkWritePair
+ */
+
+const rootDir = path.dirname(path.dirname(import.meta.url))
+const wranglerConfigPath = `${rootDir}/wrangler.toml`
+const denyListPath = `${rootDir}/denylist.json`
+
+const DENY_LIST_SOURCES = [
+  'https://badbits.dwebops.pub/denylist.json',
```

Comment:
What do you think on an ENV var for this? We could have multiple ones splitted by "," and if we automate this script we can add new lists without needing to PR

Reply:
Hmm, maybe, I'd probably make it a command line parameter like `--additional-sources` though as it's not private. There's part of me that thinks if we're going to add a new source we should probably record that in VCS though - a PR to ask the team to sign off on using a new source sounds like a good thing.